projects/GKCore/GKCore.nstd.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFrameworks>netstandard2.0</TargetFrameworks>
+        <TargetFrameworks>net6.0</TargetFrameworks>

```

Comment:
Everything, that references this file already targets .NET 6, so just switched to .NET 6, instead of having netstandard2.0 + net6.0